enlace/ajax/edit_program.php

```diff
@@ -55,10 +79,37 @@
 }
 /* change a session name: */ elseif ($_POST['action'] == 'edit_session') {
     include "../include/dbconnopen.php";
-    $new_name_sqlsafe=mysqli_real_escape_string($cnnEnlace, $_POST['new_name']);
     $id_sqlsafe=mysqli_real_escape_string($cnnEnlace, $_POST['id']);
-    $edit_session = "UPDATE Session_Names SET Session_Name='" . $_POST['new_name'] . "' WHERE Session_ID='" . $_POST['id'] . "'";
-    echo $edit_session;
+
+    $name_sqlsafe=mysqli_real_escape_string($cnnEnlace, $_POST['name']);

```

Comment:
Bugs me that this code has to be repeated verbatim like this, but I suppose you already looked for a way to abstract it out and concluded there wasn't a good way?

Reply:
Yeah, the whole code base is like this.  In general, we'd want to do some kind of ```$_POST_SAFE = make_post_safe();```, then use that.  But realistically, this is why statements and bound parameters are the right way to handle this.